src/pages/components/CertificateLabel.qml

```diff
@@ -0,0 +1,46 @@
+/****************************************************************************
+**
+** Copyright (c) 2019 Open Mobile Platform LLC.
+**
+** Contact: David Llewellyn-jones <david.llewellyn-jones@jolla.com>
```

Comment:
Drop this contact line as well...

Also would be good to amend MPL 2.0 and copyright into one comment block:

```
/* 
 * Copyright (c) 2019 Open Mobile Platform LLC.
 *
 * This Source Code Form is subject to the terms of the Mozilla Public
 * License, v. 2.0. If a copy of the MPL was not distributed with this file,
 * You can obtain one at http://mozilla.org/MPL/2.0/.
 */
```

Reply:
And same for other files as well that you have changed.

Reply:
I've merged copyright+licence text into a single block throughout (for the files that changed). I'm not entirely comfortable with dropping the attribution lines, but have done so.

Reply:
For Jolla folks attribution line dropping is ok but for contributors we need to figure a solution.

I was thinking that we add credits link to the README.md and refer to the github contributions. Also it should be noted/documented somehow that all contributors / contributions can be found from git log:

> git log --format="%aN - %ae" | sort | uniq